config/sample-input-tf-enterprise.tfvars

```diff
@@ -38,7 +38,7 @@ create_static_kubeconfig = true
 # tags = {} # e.g., { "key1" = "value1", "key2" = "value2" }
 
 # GKE config
-kubernetes_version         = "1.23.8-gke.1900"
+kubernetes_version         = "1.24"
```

Comment:
Just checking, but this works now for all 3 providers correct? That is having the k8s version listed as only the MAJOR.minor version leaving off the patch level.